src/coreComponents/physicsSolvers/multiphysics/MultiphasePoromechanics.cpp

```diff
@@ -118,6 +118,12 @@ void MultiphasePoromechanics< FLOW_SOLVER >::registerDataOnMesh( Group & meshBod
   if( m_stabilizationType == StabilizationType::Global ||
       m_stabilizationType == StabilizationType::Local )
   {
+
+    if( this->getNonlinearSolverParameters().m_couplingType == NonlinearSolverParameters::CouplingType::Sequential )
+    {
+      this->flowSolver()->enableJumpStabilization();
```

Comment:
this enables the stabilization by default for sequential run, right?

Reply:
It shouldn't since its in the if statements for Global or Local stabilization.

Reply:
I almost think that we should have it on by default t.b.h.

Reply:
Because of how it can smear interfaces, I would be hesitant to leave it on by default. But this is certainly open for discussion as more people start to make use of it.

Reply:
with the permeabilities that people usually pick for the burdens, people will likely see weird pressures but yeah... it's a tough choice.